code/FulpstationCode/sec_expanded_loadouts/detective_goggles.dm

```diff
@@ -14,8 +14,8 @@
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
 
 /obj/item/clothing/glasses/detective
-	name = "detective glasses"
-	desc = "Stylish glasses with integrated medical, diagnostic and security HUDs and reagent scanning used by detectives. The Meson Scanner mode lets you see basic structural and terrain layouts through walls. Flash proofing compromised to accomodate HUD integration."
+	name = "detective's glasses"
+	desc = "Stylish glasses with integrated medical, diagnostic and security HUDs and reagent scanning used by detectives. The Meson Scanner mode lets you see basic structural and terrain layouts through walls. WARNING! Flash proofing has been compromised to accomodate HUD integration."

```

Comment:
is it "flash proofing" or "flash-proofing"? I forget (and genuinely don't know)

Reply:
I don't think it matters.